web/satellite/static/emails/AccountAlreadyExists.html

```diff
@@ -1,599 +1,174 @@
-                                    <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml"
-                                        href="{{ .SignInLink }}" style="width:191px" arcsize="9%" fillcolor="#2683FF"
-                                        stroke="f">
-                                        <v:textbox style="mso-fit-shape-to-text:t" inset="0px,11px,0px,11px">
-                                            <center style="font-size:14px;line-height:24px;color:#FFFFFF;
-                                                font-family: Helvetica, sans-serif; font-weight:bold;
-                                                mso-line-height-rule:exactly;mso-text-raise:4px">Sign In
-                                            </center>
-                                        </v:textbox>
-                                    </v:roundrect>
-                                    <![endif]-->
-                                </div>
-                            </div>
-
-                        </div>
-                        <!--[if (mso)|(IE)]></td></tr></table><![endif]-->
-                    </div>
-                </div>
-                <div style="mso-line-height-rule: exactly;line-height: 20px;font-size: 20px;">&nbsp;</div>
-                <div class="layout one-col fixed-width"
-                    style="Margin: 0 auto;max-width: 600px;min-width: 320px;width: calc(28000% - 167400px);
-                    overflow-wrap: break-word;word-wrap: break-word;word-break: break-word;">
-                    <div class="layout__inner"
-                        style="border-collapse: collapse;display: table;width: 100%; background-color: #fff;">
-                        <!--[if (mso)|(IE)]>
-                        <table align="center" cellpadding="0" cellspacing="0" role="presentation">
-                            <tr class="layout-fixed-width" style="background-color: #fff;">
-                            <td style="width: 600px" class="w560">
-                        <![endif]-->
-                        <div class="column"
-                            style="text-align: left;color: #8e959c;font-size: 14px;line-height: 21px;
-                            font-family: Helvetica, sans-serif; font-weight: 300;max-width: 600px;min-width: 320px;width: calc(28000% - 167400px);">
-                            <div style="margin: 12px 20px;">
-                                <div style="mso-line-height-rule: exactly;mso-text-raise: 4px;">
-                                    <p class="size-20"
-                                        style="Margin-top: 5px;Margin-bottom: 0;
-                                        font-family: Helvetica, sans-serif; font-weight: 300;font-size: 17px;
-                                        line-height: 26px;" lang="x-size-20">
-                                        <span>If this wasn't you, we recommend <a href="{{ .ResetPasswordLink }}" target="_blank" rel="noopener noreferrer">resetting your password</a>.
```

Comment:
reset password link was removed in this file, causing two tests to fail. We either need to add the reset password link back to the email, or remove the checks for that link from the tests

Reply:
thanks, i just updated the template